src/typeahead/typeahead.directive.ts

```diff
@@ -344,7 +344,15 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
     this._subscriptions.push(
       this.keyUpEventEmitter
         .debounceTime(this.typeaheadWaitMs)
-        .mergeMap(() => this.typeahead)
+        .mergeMap((value: string) => {
```

Comment:
`typeahead` can be almost anything, and you would knew about it, if we would have type on it :(
so this PR will break 99% of use cases

Reply:
`value: string` is not typing for `typeahead`. This code is correct and doesn't break anything